crates/topos-tce-storage/Cargo.toml

```diff
@@ -16,16 +16,11 @@ thiserror.workspace = true
 tokio = { workspace = true, features = ["full"] }
 tokio-stream.workspace = true
 tracing.workspace = true
+once_cell.workspace = true
```

Comment:
`once_cell` doesn't need to be a workspace dependency, only the `tce-storage` is using it for now

Reply:
For now I don't think of a particular crate that could use it instead of storage. Maybe we can postpone this change when we'll be in this situation?